### ISSUE TITLE
ERR value is not an integer or out of range

### ISSUE THREAD

**Issue by mxstbr** [teamplanes/graphql-rate-limit#22]
* **I'm submitting a ...**
[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository
[ ] question about how to use this project

* **Summary**

We're hitting a bug in production that says: "ERR value is not an integer or out of range"

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

The specific redis set call gets these arguments:

```
redis-store-id::c728ce4f-2c73-435b-967d-7c851d5d752a:publishThread, 
[1548055335792], 
EX, 
600.001
```

As you can see `600.001` is not an int, but a float.

**Comment by kirkness** [comment]
Which version are you on?

**Comment by mxstbr** [comment]
Aha, apparently this was fixed already four days ago! https://github.com/teamplanes/graphql-rate-limit/commit/e046d62eb7335362565b79ff1fdc26b5bd064436

Will try upgrading, one sec

**Comment by mxstbr** [comment]
Fixed in withspectrum/spectrum#4558 by upgrading to latest, sorry for the noise!

**Comment by kirkness** [comment]
Oopsie, released in [1.2.3](https://unpkg.com/graphql-rate-limit@1.2.3/build/main/lib/redis-store.js).
